pkg/v1/providers/ytt/02_addons/management-packages/addons-manager/add_addons_manager_cr.yaml

```diff
@@ -0,0 +1,43 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:yaml", "yaml")
+
+#@ def addons_manager_default_resource():
+apiVersion: config.tanzu.vmware.com/v1alpha1
+kind: Addons-Manager
```

Comment:
question: could you help share the pointer where this CRD is defined ?

Reply:
deleted this file since we will introduce addons-manager installation & upgrade scenario in a following PR